src/init.cpp

```diff
@@ -956,6 +957,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     fIsBareMultisigStd = GetBoolArg("-permitbaremultisig", true);
     nMaxDatacarrierBytes = GetArg("-datacarriersize", nMaxDatacarrierBytes);
+    nRbfPolicy = GetArg("-replacebyfee", nRbfPolicy, 1);
```

Comment:
Yes, he added it in the other commit. The third value is the default if the value string is empty, i.e. "-replacebyfee" defaults to "-replacebyfee 1"